.editorconfig

```diff
@@ -0,0 +1,35 @@
+# editorconfig.org
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
+
+[*.java]
```

Comment:
This doesn't seem to align with the aosp format that is enforced in the builds etc.